docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -185,7 +185,7 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `FROM`: **\<empty\>**: Mail from address, RFC 5322. This can be just an email address, or
    the "Name" \<email@example.com\> format.
 - `USER`: **\<empty\>**: Username of mailing user (usually the sender's e-mail address).
-- `PASSWD`: **\<empty\>**: Password of mailing user.
+- `PASSWD`: **\`\<empty\>\`**: Password of mailing user.  Use \`your password\` for quoting, if you use special characters in the password.
```

Comment:
quoting `<empty>` is unlikely correct, it's not a password, is an indication to the user that there's NOTHING written as a password...